ui/app/components/wizard-content.js

```diff
@@ -1,13 +1,100 @@
 import { inject as service } from '@ember/service';
 import Component from '@ember/component';
+import { computed } from '@ember/object';
+import { FEATURE_MACHINE_STEPS, INIT_STEPS } from 'vault/helpers/wizard-constants';
+
 export default Component.extend({
   wizard: service(),
   classNames: ['ui-wizard'],
   glyph: null,
   headerText: null,
+  selectProgress: null,
+  currentMachine: computed.alias('wizard.currentMachine'),
+  tutorialState: computed.alias('wizard.currentState'),
+  tutorialComponent: computed.alias('wizard.tutorialComponent'),
+  showProgress: computed('wizard.featureComponent', 'tutorialState', function() {
+    return (
+      this.tutorialComponent.includes('active') &&
```

Comment:
Is this just the name of the component? why would it have 'active'?

Reply:
Yeah. It was showing the status bar during the paused state since it still had the right data in localStorage so this only shows it if we're if `tutorial-active` component is showing.

Reply:
Aha! I missed that one (I looked through the component names and didn't see it so I was confused).